Header header
uint16 device_id
uint32 distance
int16 rss
int16 anchor_x
int16 anchor_y
int16 anchor_z
